src/Sentry/SentryOptions.cs

```diff
@@ -320,6 +320,11 @@ public IDiagnosticLogger DiagnosticLogger
             }
         }
 
+        /// <summary>
+        /// Whether or not to include referenced assemblies in each event sent to sentry. Defaults to <see langword="true"/>.
+        /// </summary>
+        public virtual bool ReportAssemblies { get; set; } = true;
```

Comment:
Do you have a use-case to extend this property?
I'd rather leave it non virtual for the time being as Options passing was not designed to be extended.

Reply:
Oh yeah, not sure why I set that like that. Updated.